stl/src/xcharconv_ryu_tables.cpp

```diff
@@ -0,0 +1,4702 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

```

Comment:
Ryu-derived code needs special treatment. This correctly mentions `ryu` in the filename, but we also need to preserve the banner. We should have the usual banner followed by the Ryu banner, exactly like this:

https://github.com/microsoft/STL/blob/a9321cfe53ea31a7e197d5d8336167d6ca3de8b6/stl/inc/xcharconv_ryu_tables.h#L3-L32